lib/tether/tether.go

```diff
@@ -580,10 +584,14 @@ func (t *tether) launch(session *SessionConfig) error {
 		extraconfig.EncodeWithPrefix(t.sink, session, prefix)
 	}()
 
-	// Special case here because UID/GID lookup need to be done
-	// on the appliance...
-	if len(session.User) > 0 {
-		session.Cmd.SysProcAttr = getUserSysProcAttr(session.User)
+	if len(session.User) > 0 || len(session.Group) > 0 {
+		user, err := getUserSysProcAttr(session.User, session.Group)
+		if err != nil {
+			log.Errorf("user lookup failed %s:%s, %s", session.User, session.Group, err)
```

Comment:
Now the error message of docker command looks like this
```docker: Error response from daemon: Server error from portlayer: Unable to find user nonexist.```
do I need to add more information to it?
For the log itself, feels like a bit more things should be okay